lib/job-iteration.rb

```diff
@@ -11,6 +13,10 @@ module JobIteration
 
   extend self
 
+  attr_accessor :logger
+
+  self.logger = ActiveJob::Base.logger
```

Comment:
This does change something. Previously we ended up logging to `Rails.logger`, because when ActiveJob initializes itself in the context of a Rails application, it sets its logger to the Rails one:
https://github.com/rails/rails/blob/v7.0.7.2/activejob/lib/active_job/railtie.rb#L13-L15

Whereas now it will use the default Active Job logger which logs to stdout: https://github.com/rails/rails/blob/v7.0.7.2/activejob/lib/active_job/logging.rb#L11

Reply:
TIL. Thanks for also providing a fix - linking https://github.com/Shopify/job-iteration/pull/427 here in case others run into this problem.